Jint/Native/JsSet.cs

```diff
@@ -47,17 +47,17 @@ protected override bool TryGetProperty(JsValue property, [NotNullWhen(true)] out
         return base.TryGetProperty(property, out descriptor);
     }
 
-    internal void Add(JsValue value) => _set.Add(value);
+    public void Add(JsValue value) => _set.Add(value);
 
-    internal void Remove(JsValue value) => _set.Remove(value);
+    public void Remove(JsValue value) => _set.Remove(value);
```

Comment:
I think this should be just plain `Delete`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
You mean rename the method itself to `Delete(JsValue value)`? Given that it was previously internal, am I safe to assume that all existing consumers that could break as a result are in the same repo? Hopefully refactoring it globally is adequate.

Reply:
OK, so I see it's only used in SetPrototype, however I wonder if the name "SetDelete" was used to avoid conflicting with ObjectInstance.Delete() which would attempt to delete a property by that name from the object (presumably like javascript's `delete foo.bar`) whereas we're wanting `mySet.delete("abc")` to take a value out of the set.